clients-interpreted.yml

```diff
@@ -174,3 +174,15 @@
       - project:
             level: 3
             feature: [beetmover-get-artifact-scope]
+
+# Clients for landoscript workers
+- client:
+      project/releng/scriptworker/v2/lando/dev/firefoxci-{trust_domain}-1:
+          description: "{trust_domain} level 1 nonprod lando scriptworker"
+          scopes:
+              - queue:claim-work:scriptworker-k8s/{trust_domain}-1-lando-dev
+              - queue:worker-id:{trust_domain}-1-lando-dev/{trust_domain}-1-lando-dev-*
```

Comment:
So sharing a single client across projects is harmless as long as the scopes don't have `{trust_project}` anywhere in them. As it stands, even if we did split clients out by project they would all have the same scopes. So there would be no security benefit of doing that.

However, maybe there should be per-project scopes? I guess it ultimately comes down to Lando. Will they be providing us with a single access token that can land to every branch in the new repo? Or will there be per-branch tokens?

If it's the former, then a single scope is fine. If it's the latter, we might want to separate it out here as well.

Reply:
> However, maybe there should be per-project scopes? I guess it ultimately comes down to Lando. Will they be providing us with a single access token that can land to every branch in the new repo? Or will there be per-branch tokens?

The token or tokens we'll be getting are unclear at this time. It's also unclear if we'll have extra scopes that may differ by project.

However...we will need separate workloads per trust domain as I understand things, unless you're suggesting that we have a single `landoscript` workload that doesn't include trust domain anywhere? (I'm not even sure how that would work for chain of trust verification though...)

Obviously we could share the same client id across multiple workloads; I'm not sure if that's a good, bad, or neutral thing...